configuration/concepts/surgery_checklist_concepts.csv

```diff
@@ -0,0 +1,77 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Data class,Data type,Answers,Members,_version:1:,_order:1500
```

Comment:
@PiusKariuki , the order number is relatively high, especially since we might reuse these concepts in other forms created in the future. I'll adjust it to something like 40, as it should typically be just above the last created form.

Reply:
@PiusKariuki , a question coded answer usually has a Misc / NA and not a Misc / NA.

Reply:
@daviemukungi Can you please clarify this last comment I think there's a type of something because the issue and the fix are the same "Misc / NA"